### PR TITLE
Remove duplicate env var constant for AFL_DRIVER_DONT_DEFER

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

File: src/python/bot/fuzzers/afl/launcher.py [hunk 1]
```diff
@@ -899,7 +899,7 @@ def check_error_and_log(error_regex, log_message_format):
 
           elif (num_first_testcase_hangs ==
                 self.MAX_FIRST_HANGS_WITH_DEFERRED_FORKSERVER):
-            environment.set_value(constants.DONT_DEFER_FORKSERVER, 1)
+            environment.set_value(constants.AFL_DRIVER_DONT_DEFER, 1)
```
Comment (inferno-chromium) [inline src/python/bot/fuzzers/afl/launcher.py:902]: I think this is still DONT_DEFER_ENV_VAR ?

Comment (jonathanmetzman) [reply src/python/bot/fuzzers/afl/launcher.py]: Oh wow, I messed this PR up. Good catch.

Comment (jonathanmetzman) [reply src/python/bot/fuzzers/afl/launcher.py]: Fixed.

**Comment by inferno-chromium** [comment]
Minor unrelated lint error, but still good to fix to let gcbrun complete.

**Comment by jonathanmetzman** [comment]
> Minor unrelated lint error, but still good to fix to let gcbrun complete.

GCB wasn't red only travis was. Weird. Does gcb not run lint/format and only travis does this?

**Comment by inferno-chromium** [comment]
> > Minor unrelated lint error, but still good to fix to let gcbrun complete.
> 
> GCB wasn't red only travis was. Weird. Does gcb not run lint/format and only travis does this?

Looks like we missed to add that (since there were lint failures in python 3). we have to fix them anyway, can you add it in cloudbuild.yaml - pipenv run python butler.py lint ?

**Comment by oliverchang** [comment]
> > > Minor unrelated lint error, but still good to fix to let gcbrun complete.
> > 
> > 
> > GCB wasn't red only travis was. Weird. Does gcb not run lint/format and only travis does this?
> 
> Looks like we missed to add that (since there were lint failures in python 3). we have to fix them anyway, can you add it in cloudbuild.yaml - pipenv run python butler.py lint ?

Let's add this after all bots are migrated to Python 3. There are a number of lint failures that are specific to Py3.

**Comment by jonathanmetzman** [comment]
> > > > Minor unrelated lint error, but still good to fix to let gcbrun complete.
> > > 
> > > 
> > > GCB wasn't red only travis was. Weird. Does gcb not run lint/format and only travis does this?
> > 
> > 
> > Looks like we missed to add that (since there were lint failures in python 3). we have to fix them anyway, can you add it in cloudbuild.yaml - pipenv run python butler.py lint ?
> 
> Let's add this after all bots are migrated to Python 3. There are a number of lint failures that are specific to Py3.

SGTM. Merging.

**Pull request merged**